src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpIntelliSense.cs

```diff
@@ -285,23 +328,27 @@ class Class1
         [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
         public void EnsureTheCaretIsVisibleAfterALongEdit()
         {
-            SetUpEditor(@"
+            var visibleColumns = VisualStudio.Editor.GetVisibleColumnCount();
+            var variableName = new string('a', (int)(0.75 * visibleColumns));
```

Comment:
📝 This test was passing for me locally because the final text all fit in the view without scrolling.

Reply:
➡️ #33822